subprojects/ide/src/main/java/org/gradle/plugins/ide/idea/IdeaPlugin.java

```diff
@@ -302,11 +302,11 @@ public File call() {
         });
 
         Set<File> excludeDirs = Sets.newLinkedHashSet();
+        excludeDirs.add(project.file(".gradle"));
```

Comment:
This changes behavior, evaluating the build directory early.

Reply:
D'oh your right of course, there is even a test for that.
Bad me first create the PR and then kicked off the tests.
Should be better now hopefully, thanks.